bin/deploy.sh

```diff
@@ -54,7 +54,8 @@ case "$ENV" in
   exit 1
 esac
 
-COMMIT=`git rev-parse --short HEAD`
+ABBREV_LENGTH=`node ${DIRNAME}/config-abbrev --env=${ENV}`
```

Comment:
As you said, I also don't think this is the ideal approach. Nevertheless it gets the job done and that's what's important here. We need to revisit this whole JS/Bash interop soon anyway.